c_glib/test/test-array.rb

```diff
@@ -131,4 +131,31 @@ def test_invalid
       assert_equal(message, error.message.lines.first.chomp)
     end
   end
+
+  sub_test_case("#diff_unified") do
+    def test_no_diff
+      array = build_string_array(["Start", "Shutdown", "Reboot"])
+      other_array = build_string_array(["Start", "Shutdown", "Reboot"])
+      assert_equal(nil, array.diff_unified(other_array))
```

Comment:
`assert_nil(array.diff_unified(other_array))` is better.

Reply:
Thanks, fixed now.